tidb-cloud/tidb-cloud-clinic.md

```diff
@@ -0,0 +1,109 @@
+---
+title: TiDB Cloud Clinic
+summary: Learn how to use advanced monitoring and diagnostics of TiDB Cloud using TiDB Cloud Clinic.
+---
+
+# TiDB Cloud Clinic
+
+TiDB Cloud Clinic offers advanced monitoring and diagnostic capabilities for TiDB Cloud.
+
+> **Note:**
+>
+> Currently, TiDB Cloud Clinic is only available for [TiDB Cloud Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-cloud-dedicated) clusters.
+
+## Prerequisites
+
+To use TiDB Cloud Clinic, you need to have an **Enterprise** support plan or higher for TiDB Cloud.
+
+## View the **Cluster** page
+
+To view the **Cluster** page, take the following steps:
+
+1. Log in to the [TiDB Cloud Clinic console](https://clinic.pingcap.com/) and select **Continue with TiDB Account** to enter the TiDB Cloud login page.
+
+2. From the organization list, select your target organization. The clusters in the selected project are displayed.
```

Comment:
It seems that users have to enter and search the organization. 
![image](https://github.com/user-attachments/assets/b5370aee-c389-4fe5-9d2c-195455c1d2fe)